Pilot3/P3B5/darts/genotypes.py

```diff
@@ -16,6 +16,16 @@
 ]
 
```

Comment:
Primitives could be moved away from the genotypes file. This might make things easier to find.